node-manager/src/localstorage.rs

```diff
@@ -85,6 +87,36 @@ impl MutinyBrowserStorage {
         map
     }
 
+    pub(crate) fn get_data_with_prefix<T: Serializable>(
```

Comment:
how is this different than `scan`?

Reply:
scan checks the key, this checks the value